concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -127,160 +191,123 @@ writeSection ::
     BlockHash ->
     -- |Height of first block in section
     BlockHeight ->
-    -- |Handle to write to
+    -- |Export directory
+    FilePath ->
+    -- |Index file handle
     Handle ->
-    -- |Action to write out the blocks and return the number of them
-    m Word64 ->
-    -- |Action to write out the finalization records and return the number of them
-    m Word64 ->
+    -- |Chunk size
+    Word64 ->
     m ()
-writeSection
+writeChunks
     sectionGenesisIndex
     sectionProtocolVersion
     sectionGenesisHash
     sectionFirstBlockHeight
-    hdl
-    writeBlocks
-    writeFinRecs = do
+    outDir indexHdl chunkSize = do
+        let chunkName = outDir </> "blocks-"
+                        ++ show sectionGenesisIndex ++ "-"
+                        ++ show sectionFirstBlockHeight
+                        ++ ".dat"
+        chunkHdl <- liftIO $ initialHandle chunkName
+
         (sectionStart, blocksStart) <- liftIO $ do
-            sectionStart <- hTell hdl
+            BS.hPut chunkHdl $ encode (3 :: Version)
```

Comment:
What is this 3 for? That was the version for the previous export. Why are we keeping the version the same? While the format seems to be the same, sections now don't start with first non-genesis blocks, so there is a difference.